database/seeds/DirectionTableSeeder.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use Faker\Factory as Faker;
+use App\Models\Directions;
+class DirectionTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $faker = Faker::create();
+    	$route_ids = factory(App\Models\Route::class, 10)->create()->pluck('id');
```

Comment:
for routes and stops, just get the existed records in DB, no need to create new records.